app/utils/query/parser.js

```diff
@@ -86,8 +86,18 @@ export default {
      * @private
      */
     evalRule:function(rule){
+        let rangeOperator = ['between','not_between'];
         if (rule.type == 'date'){
-            rule.value = moment(rule.value,'MM/DD/YYYY').format('YYYY-MM-DD');
+            if(rangeOperator.includes(rule.operator)){
+                rule.value = rule.value.map(function(value){
+                    return moment(value,'MM/DD/YYYY').format('YYYY-MM-DD');
```

Comment:
Lets change one thing. Please create a getter function called getDateDisplayFormat in either a new class Localization or in the App class, this will help us in the future. 

Would be great if we have the localization class.

Reply:
I'll handle it in a separate issue. That issue is part of this sprint: